### PR TITLE
Small fix to avoid NullPointerException

### PR DESCRIPTION
**Pull request by marcelrv**
as .equals(null) is not a correct check
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/MaxCubeBinding.java [hunk 1]
```diff
@@ -334,7 +334,7 @@ public void internalReceiveCommand(String itemName, Command command) {
 		for (MaxCubeBindingProvider provider : providers) {
 			serialNumber = provider.getSerialNumber(itemName);
 
-			if (serialNumber.equals(null)) {
+			if (serialNumber==null || serialNumber.isEmpty()) {
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.maxcube/src/main/java/org/openhab/binding/maxcube/internal/MaxCubeBinding.java]: please use StringUtils.isBlank() instead

**Comment by teichsta** [comment]
Hi @marcelrv thanks for this Fix. Please find my comment and finally please squash your commits into one. Thanks, Thomas E.-E.

**Comment by marcelrv** [comment]
Hi @teichsta ,
Got it.

**Comment by teichsta** [comment]
> Got it.

not quite :-) isBlank() already contains the "null check" …

**Comment by marcelrv** [comment]
ah.. did not know that :-(
Okay.. hope that _now_ I got it :-)

**Comment by teichsta** [comment]
yep, thanks!

**Pull request merged**